blog-homepage/src/AuthorBio.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+
+
+
+export default class AuthorBio extends React.Component{
+    render(){
+        const isMediumMember= this.props.isMediumMember;
+
+        
+        return(
+            
+                <div className = "author_details">
```

Comment:
```suggestion
                <div className = "author-details">
```
Typically html class names are `dash-separated`